api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddOutreachProviderRoleToPersonGenerator.java

```diff
@@ -20,127 +20,127 @@ String getTemplate() {
+        //            "http://purl.org/NET/c4dm/event.owl#Event","Event",
+        //            "http://vivoweb.org/ontology/core#EventSeries", "Event Series",
+        //            "http://vivoweb.org/ontology/core#Exhibit", "Exhibit",
+        //            "http://vivoweb.org/ontology/core#ExtensionUnit","Extension Unit",
+        //            "http://vivoweb.org/ontology/core#Foundation","Foundation",
+        //            "http://vivoweb.org/ontology/core#FundingOrganization","Funding Organization",
+        //            "http://vivoweb.org/ontology/core#GovernmentAgency","Government Agency",
+        //            "http://xmlns.com/foaf/0.1/Group","Group",
+        //            "http://purl.org/ontology/bibo/Hearing", "Hearing",
+        //            "http://vivoweb.org/ontology/core#Hospital","Hospital",
+        //            "http://vivoweb.org/ontology/core#Institute","Institute",
+        //            "http://purl.org/ontology/bibo/Interview", "Interview",
+        //            "http://vivoweb.org/ontology/core#InvitedTalk", "Invited Talk",
+        //            "http://vivoweb.org/ontology/core#Laboratory","Laboratory",
+        //            "http://vivoweb.org/ontology/core#Library","Library",
+        //            "http://purl.obolibrary.org/obo/OBI_0000835","Manufacturer",
+        //            "http://vivoweb.org/ontology/core#Meeting", "Meeting",
+        //            "http://vivoweb.org/ontology/core#Museum","Museum",
+        //            "http://xmlns.com/foaf/0.1/Organization","Organization",
+        //            "http://purl.org/ontology/bibo/Performance", "Performance",
+        //            "http://vivoweb.org/ontology/core#Presentation", "Presentation",
+        //            "http://vivoweb.org/ontology/core#PrivateCompany","Private Company",
+        //            "http://vivoweb.org/ontology/core#Program","Program",
+        //            "http://vivoweb.org/ontology/core#Publisher","Publisher",
+        //            "http://vivoweb.org/ontology/core#ResearchOrganization","Research Organization",
+        //            "http://vivoweb.org/ontology/core#School","School",
+        //            "http://vivoweb.org/ontology/core#SeminarSeries", "Seminar Series",
+        //            "http://vivoweb.org/ontology/core#Team","Team",
+        //            "http://vivoweb.org/ontology/core#ServiceProvidingLaboratory","Service Providing Lab",
+        //            "http://vivoweb.org/ontology/core#StudentOrganization","Student Organization",
+        //            "http://purl.obolibrary.org/obo/ERO_0000565","Technology Transfer Office",
+        //            "http://vivoweb.org/ontology/core#University","University",
+        //            "http://purl.org/ontology/bibo/Workshop", "Workshop",
+        //            "http://vivoweb.org/ontology/core#WorkshopSeries", "Workshop Series");
 	}
 
 	@Override
 	boolean isShowRoleLabelField(){return true;}
 
-	/*
```

Comment:
I would have expected that this pull-request only include the updates from here (line 136/137) to the bottom. The other changes above are all functional changes that Michel made, and do not need reformating.

How were you viewing the required updates?

Reply:
... obviously in a wrong manner. Sorry, I'm still struggling to diff my-whitespace to sprint-i18n-whitespace, and my-whitespace to master. This new push should be ok, though.